lib/menu.rb

```diff
@@ -0,0 +1,21 @@
+class Menu
+
+  attr_reader :food_list
+
+  def initialize
+    @food_list = {'Rainbow dumpling' => 28, 'Half crispy aromatic duck' => 39}
```

Comment:
Aside from just the hardcoding, it's not immediately apparent to me what the integer refers to - is it quantity or price? As the list is a hash I have no specifications to refer back to